### ISSUE TITLE
Object not found in the ci.spdk.io web for both nightly and per patch log file

### ISSUE THREAD

**Issue by wanqunintel** [spdk/spdk#552]
Object not found!
The requested URL was not found on this server. The link on the referring page seems to be wrong or outdated. Please inform the author of that page about the error.

If you think this is a server error, please contact the webmaster.

Error 404
ci.spdk.io
Apache
https://ci.spdk.io/spdk-jenkins/results/autotest-per-patch/builds/17812/archive/vhost_autotest/

**Comment by jimharris** [comment]
It seems there are a lot of these.  For example - Jenkins votes this patch +1 over 9 hours ago:

https://review.gerrithub.io/#/c/spdk/spdk/+/437574/
https://ci.spdk.io/spdk-jenkins/public_build/autotest-per-patch_17869.html

This patch was voted +1 a couple of hours earlier than that:

https://review.gerrithub.io/#/c/spdk/spdk/+/437309/

It does not appear any of the result files are available from today.

**Comment by peluse** [comment]
Yup I just ran into one myself and started looking over the post processing scripts.   just sent Karol & Seth a note, we will all look into this together and see how we can beef up both the process and the debug output.

**Comment by karlatec** [comment]
Looking into the issue with Pawel K, thanks for notifying.

**Comment by Seth5141** [comment]
Here are a couple of interesting failures from last night

1. Has the top level directory, no sub directories at all. https://ci.spdk.io/spdk-jenkins/public_build/autotest-per-patch_18043.html

2. Has the top level directory, only failed subjobs missing. https://ci.spdk.io/spdk-jenkins/public_build/autotest-per-patch_18034.html

**Comment by wanqunintel** [comment]
Current status:
1. For failure jobs, the log for the failed test still have the "object not found" issue. https://ci.spdk.io/spdk-jenkins/public_build/autotest-per-patch_18487.html
2. For the manual run jobs, the log still not found
3. For nightly jobs, it all failed with the errors as follow 
"LINK test/unit/lib/bdev/mt/bdev.c/bdev_ut
Build was aborted
Aborted by unknown"
https://ci.spdk.io/spdk-jenkins/public_build/autotest-per-patch_18445.html

**Comment by karlatec** [comment]
Hi Anna!
1. Do you check the logs immediately after the build has finished? You should give it some time (couple of minutes at least) because we have to sync the logs to the server. Working on speeding things up.
2. What do you mean by manual run jobs? Does that apply to autotest-per-patch or some other job?

**Comment by darsto** [comment]
This is work in progress. @peluse will give us an update soon
